draco/analysis/transform.py

```diff
@@ -610,6 +641,19 @@ def process(self, mmodes: containers.MContainer) -> containers.SiderealContainer
         nra_cont = 2 * mmodes.mmax + (1 if mmodes.oddra else 0)
         nra = self.nra if self.nra is not None else nra_cont
 
+        # Apply the m-mode sinc-suppression caused by the rectangular integration window
+        if self.apply_integration_window:
+
+            m = np.arange(mmodes.mmax + 1)
+            w = np.sinc(m / nra)
+            inv_w = tools.invert_no_zero(w)
+
+            sl_vis = (slice(None),) + (np.newaxis,) * (len(mmodes.vis.shape) - 1)
+            mmodes.vis[:] *= w[sl_vis]
+
+            sl_weight = (slice(None),) + (np.newaxis,) * (len(mmodes.vis.shape) - 1)
```

Comment:
Same comment as above
```suggestion
            sl_weight = (slice(None),) + (np.newaxis,) * (len(mmodes.weight.shape) - 1)
```